spec/api-browser-window-spec.js

```diff
@@ -86,6 +86,14 @@ describe('BrowserWindow module', () => {
     return closeWindow(w).then(() => { w = null })
   })
 
+  describe('BrowserWindow constructor', () => {
+    it('allows passing void 0 as the webContents', () => {
+      w = new BrowserWindow({
+        webContents: void 0
```

Comment:
Can we simply use `undefined` here? )
I have never seen the `void` operator being used and I guess a lot of people don't  know it even exists )

Reply:
`undefined` didn't trigger the failure in this exact case, I'm assuming because we have an undefined check somewhere else 🤔 `void 0` triggered it everytime so I just left that in the test

Reply:
That's weird. [`void` operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/void) simply returns the `undefined` value for any expression it's used with. There cannot be any difference.